main.py

```diff
@@ -335,16 +342,25 @@ def play(self):
         else:
             winsound.PlaySound(None, winsound.SND_PURGE)
             self.musicPlaying = False
+        logging.debug('end of play')
 
+def showMessageBox( title, message):
+    messagebox.showinfo(title, message)
 
+def initLogging():
+    FORMAT = "%(asctime)s — %(name)s — %(levelname)s — %(funcName)s:%(lineno)d — %(message)s"
+    logging.basicConfig(format=FORMAT)
+    logging.getLogger().setLevel(logging.DEBUG)
 
 if __name__ == '__main__':
-
+    initLogging()
     # wave_obj = sa.WaveObject.from_wave_file(filename)
     # play_obj = wave_obj.play()
     g = game()
-
+    g.t_start = time.time()
```

Comment:
maybe not needed since we have self.t_start in run() ?

Reply:
for some reason the t_start is not initialized 
    minutes = (t_end - self.t_start) // 60
                       ^^^^^^^^^^^^
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.496.0_x64__qbz5n2kfra8p0\Lib\tkinter\__init__.py", line 2425, in __getattr__
    return getattr(self.tk, attr)
           ^^^^^^^^^^^^^^^^^^^^^^
AttributeError: '_tkinter.tkapp' object has no attribute 't_start'
adding this line was able to fix the bug.